File: Pset6 - Python/sentimental-mario-less/mario.py
from cs50 import get_int

# variable for number of blocks
while True:
    num_blocks = get_int("How high should the pyramid be? ")
    if num_blocks > 0 and num_blocks < 9:
        break

# number of blocks must be between 0 and 9
for i in range(1, num_blocks + 1):
    spaces = num_blocks - i

    print(" " * spaces, end="")
    print("#" * i)

File: Pset1 - C/mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //variable for height integer in a do while loop
    int n;
    do
    {
        n = get_int("How tall do you want your pyramid to be? (It can be 1 to 8 bricks high) ");
    }
    while (n < 1 || n > 8);  // height can be greater than 0 and less than 9 (1 - 8)


    // nested loops for printing "bricks" --> #
    for (int i = 0; i < n; i++)  // rows
    {
        for (int k = 0; k < (n - (i + 1)); k++) // additional for loop to account for spaces and the right aligned pyramid format
        {
            printf(" ");

        }
        for (int j = 0; j < (i + 1); j++)  // columns
        {

            printf("#");

        }

        // move to next line
        printf("\n");

    }

}
